php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -855,10 +847,6 @@ class NDB_Menu_Filter extends NDB_Menu
         }
         $query .= $this->order_by;
 
-        if (!empty($this->limit)) {
```

Comment:
I couldn't find any modules making use of this either.

Reply:
(if there were, it should cause phan to fail since this PR is enabling phan detecting undefined property accesses..)